src/SimpleDetailsList.tsx

```diff
@@ -1,21 +1,23 @@
 import * as React from 'react';
+import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
+import {
+  ContextualMenu,
+  DirectionalHint,
+  IContextualMenuItem,
+  IContextualMenuProps
+} from 'office-ui-fabric-react/lib/ContextualMenu';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';

```

Comment:
it would be better to just bundle the things you need; using the uber bundle is bound to include a ton of stuff you don't want. sent mail thread separately.